configure.ac

```diff
@@ -61,9 +61,10 @@ AC_SUBST([LIBOPENARC_VERSION_INFO])
 # Checks for programs
 #
 AC_PROG_CC
-AC_PROG_CC_C99
+# AC_PROG_CC_C99
 AM_PROG_CC_C_O
-AC_PROG_LIBTOOL
+# AC_PROG_LIBTOOL
+LT_INIT
```

Comment:
These changes might be correct, but the obsolete macros should not be left as comments, they should just be removed.

Since it's no longer a check for a program, `LT_INIT` should probably be relocated to the autotools setup section.

Reply:
I removed, rather than commented out, the obsolete macros that gave errors when I tried to create a configure file.

I left the LT_INIT as I was not sure where else to put it.